src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -751,7 +751,9 @@ void TransportAdapterImpl::ConnectFailed(const DeviceUID& device_handle,
 void TransportAdapterImpl::RemoveFinalizedConnection(
     const DeviceUID& device_handle, const ApplicationHandle& app_handle) {
   const DeviceUID device_uid = device_handle;
+#if ENABLE_LOG
   const ApplicationHandle app_uid = app_handle;
+#endif
```

Comment:
You can actually get around macroing this out by changing Line 760 from:
```cpp
connections_.find(std::make_pair(device_uid, app_handle));
```
to 
```cpp
connections_.find(std::make_pair(device_uid, app_uid));
```
Based on other functions in this file with similar variables, this is how the variable should be used